tmf2/org.lttng.scope.tmf2.views.ui.tests/src/org/lttng/scope/tmf2/views/ui/timeline/widgets/timegraph/TimeGraphWidgetSeekTest.java

```diff
@@ -133,39 +133,43 @@ public void testZoomOutEnd() {
      */
     @Test
     public void testZoomOutFull() {
-        testZoom(160000L, 200000L, true, 1);
-        testZoom(180000L, 200000L, true, 3);
+        testZoom(160000L, 200000L, 180000L, true, 1);
+        testZoom(180000L, 200000L, 190000L, true, 3);
```

Comment:
Would it be worth changing/adding tests with a pivot that is not in the exact center of the visible range?